src/main/java/seedu/address/model/powercard/Question.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.powercard;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Powercard's Question in the deck.
+ */
+public class Question {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Questions should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
```

Comment:
I think we can modify the comment from "address" to "question" in case we miss it out in the future